html/janus.js

```diff
@@ -1429,13 +1438,12 @@ function Janus(gatewayCallbacks) {
 			// If we got here, we're not screensharing
 			if(media === null || media === undefined || media.video !== 'screen') {
 				// Check whether all media sources are actually available or not
-				// as per https://github.com/meetecho/janus-gateway/pull/114
-				MediaStreamTrack.getSources(function(sources) {
-					var audioExist = sources.some(function(source) {
-						return source.kind === 'audio';
+				navigator.mediaDevices.enumerateDevices().then(function(devices) {
```

Comment:
See above... are you updating adapter.js? I can't see the changes. Are we risking putting browser specific stuff in here that will work, e.g., for Chrome but not Firefox?